pkg/operatorclient/operatorclient.go

```diff
@@ -95,3 +95,52 @@ func (c OperatorClient) SetObjectAnnotations(required map[string]string) error {
 	}
 	return nil
 }
+
+func (c OperatorClient) EnsureFinalizer(ctx context.Context, finalizer string) error {
```

Comment:
Any reasons we're not using the generic operator client here?
Regardless, not for this PR.

Reply:
I tried to use generic operator client from https://github.com/openshift/library-go/blob/master/pkg/operator/genericoperatorclient/dynamic_operator_client.go, but it does not support `SetObjectAnnotations()` used in [OLMOperatorRemovalController](https://github.com/openshift/cluster-storage-operator/blob/53afa77f6bbca7cfe28b914930beff2b3c407d93/pkg/operator/csidriveroperator/olmremovalcontroller.go#L262)

Removing `OLMOperatorRemovalController` could be a way forward.